### DISCUSSION TITLE
[Simulation question]: Four General Questions about the G4-DNA physics and Topas-nBio

### DISCUSSION THREAD

**Discussion by ShahinGHD** [topas-nbio/TOPAS-nBio#24] [Simulation-question] [answered]
### Simulation question

Hi everyone, I have these four questions about G4-DNA and TOPAS. I appreciate your clarification.

1- To measure the DNA breaks from radionuclides using DBSCAN, is it better to use the g4em-dna (default) or opt6? Because papers showing opt6  is closer to other codes results but I'm seeing sometimes the results are realy different from g4em-dna physic.

2- Is option 6 actually able to simulate the radiolysis of water in the DBSCAN? or do we need to include any chemistry in the physics list?
or is not possible to have that in DBSCAN.

3- Is the repair also taken into account in this module? Or do we have to account for it after we get the results?

4- Is it possible to define two spheres one inside each other and have an array of those repeated in a phantom in TOPAS-nBio? For example 200x200x200 of those with some intervals.

I thank Jose for answering my previous questions.
Thank you for your time

Shahin

### TOPAS-nBio category

Physics

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by JoseRamosMendez** [comment, marked as answer]
Hi Shahin, regarding your questions

1. Those physics options might be different for electrons at the lowest energies (within a few tens of keV). For releasing TOPAS-nBio, we performed a set of regression tests in which we included the physics models comparison for DBSCAN, see the PDF file available in [this repository](https://github.com/topas-nbio/TOPAS-nBio-regression.git). Use those results to drive your decision of which model is good for your research. We recommend using ["TsEmDNAPhysics"](https://topas-nbio.readthedocs.io/en/latest/Processes/Physics.html), which is customizable and was used for validating TOPAS-nBio, but you're free to use Option-6.
2.  All physics options can be used to simulate the radiolysis of water, but you must include the chemistry reactions. I recommend you to surf through the TOPAS-nBio examples, see e.g., [ActiveChemistryRevised.txt](https://github.com/topas-nbio/TOPAS-nBio/blob/767d44481c374d35e47449b0c0923f7f5c92b7ae/examples/processes/ActiveChemistryRevised.txt)
2a. The current DBSCAN implementation **only supports physics interactions**, not contributions from radiolytic species
3. Repair is not included. But TOPAS-nBio has models via Damaris and MEDRAS **Please check this section in the** [**documentation**](https://topas-nbio.readthedocs.io/en/latest/Modeling/DNARepair.html).
4. Yes it is possible, but it requires a new geometry implementation. 

Cheers

**Reply by ShahinGHD** [reply]
Thank you Jose, You answered all my questions. I appreciate it
I think the reason I need to use g4em-dna is that I need to use it with "AssignToRegionNamed". It was a choice between the default version and opt6. I am just doing some tests to see which one is better, opt4 seems to be more reasonable based on the regression docs. its very close to Francis 2011 paper and Partrac

Thanks again
